jsonschema-hyperschema.xml

```diff
@@ -679,8 +679,8 @@
                 </t>
                 <figure>
                     <preamble>
-                        Recall that a Relative JSON Pointer which is not also a regular
```

Comment:
Ugh, neither of these is correct.  regular JSON Pointers are also valid Relative JSON Pointers.  I had it backwards.  Yours is correct but not what I meant :-P  I'll fix it and update the main PR (possibly by merging this as-is and then amending it, need to read the paragraph re-write below and think on it).

Reply:
Although really there's some debate over this.  Some people seem to think that it's necessar to say "JSON Pointer or Relative JSON Pointer" which seems horribly verbose to me.  Not sure what to do with it.  I was just going to change the Relative JSON Pointer spec to say what I want it to say and re-submit it as draft-andrews-relative-json-pointer-00, probably :-P

Reply:
Yes this wasn't clear for me. The approach you took in #386 is better, thanks.